include/deal.II/lac/petsc_ts.h

```diff
@@ -441,10 +441,9 @@ namespace PETScWrappers
      * @note This variable represents a
      * @ref GlossUserProvidedCallBack "user provided callback".
      * See there for a description of how to deal with errors and other
-     * requirements and conventions. PETSc's TS package can not deal
-     * with "recoverable" errors, so if a callback
-     * throws an exception of type RecoverableUserCallbackError, then this
-     * exception is treated like any other exception.
+     * requirements and conventions.
+     * The recovery callback can be customized using
+     * TimeStepper::recoverable_action_function.
```

Comment:
This is no longer the case (and the same for the other places below where you have similar text in the documentation).

Reply:
thanks for double checking, I will remove this